system-contracts/contracts/precompiles/CodeOracle.yul

```diff
@@ -88,21 +88,21 @@ object "CodeOracle" {
                     // Decommitment failed
                     revert(0,0)
                 }
-                
+
                 // The "real" result of the `decommit` operation is a pointer to the memory page where the data was unpacked.
                 // We do not know whether the data was unpacked into the memory of this contract or not.
                 //  
                 // Also, for technical reasons we can not access pointers directly, so we have to copy the pointer returned by the
                 // decommit operation into the `active` pointer. 
                 verbatim_0i_0o("decommit_ptr_to_active")
 
-                // This operation is never expected to overflow since the `lenInWords` is at most 2 bytes long.
+                // This operation is never expected to overflow since the `lenInWords` is a most 2 bytes long.
                 let lenInBytes := mul(lenInWords, 32) 
 
                 // To avoid the complexity of calculating the length of the preimage in circuits, the length of the pointer is always fixed to 2^21 bytes.
                 // So the amount of data actually copied is determined here.
                 // Note, that here we overwrite the first `lenInBytes` bytes of the memory, but it is fine since the written values are equivalent
-                // to the bytes previously written there by the `decommit` operation (in case this is the first page where the decommit happened).
+                // to the bytes previously written there by the `decommit` operation (in case this is the first page where the decomit happened).
```

Comment:
```suggestion
                // to the bytes previously written there by the `decommit` operation (in case this is the first page where the decommit happened).
```